package.json

```diff
@@ -5,7 +5,11 @@
   "main": "lib/Slider",
   "jsnext:main": "src/Slider.jsx",
   "scripts": {
-    "build": "npm run clean && babel src -d lib",
+    "prebuild": "npm run clean",
+    "build": "npm run build:js && npm run build:css",
+    "build:js": "babel src -d lib",
+    "prebuild:css": "rimraf lib/css && mkdir -p lib/css",
+    "build:css": "node scripts/buildCSS.js",
     "clean": "rimraf lib",
     "prepublish": "in-publish && safe-publish-latest && npm run build || not-in-publish",
     "lint": "eslint --ext .js,.jsx src test stories",
```

Comment:
Do we want to add a script for running the css storybook? e.g. 
```    
"storybook:css": "npm run build:css && start-storybook -p 9001 -c .storybook-css",
```

Reply:
Have this in a later commit! https://github.com/airbnb/rheostat/pull/166